code/drasil-example/Drasil/NoPCM/Body.hs

```diff
@@ -199,6 +202,7 @@ si = SI {
   _instModels  = NoPCM.iMods,
   _datadefs    = NoPCM.dataDefs,
   _configFiles = [],
+  _folderPath  = directoryName,
```

Comment:
@JacquesCarette, the changes requested in #2669 have been sort-of resolved here. 

I manually had to add a `directoryName` for examples like NoPCM and DblPendulum, as the system's abbreviation is different from its folder path. I know that the pendulum example is currently being in the process of splitting into single and double examples, but NoPCM directly uses the same system name as SWHS, so I would have to manually add it as a string here. For the rest of the examples, I could grab their abbreviation and take away the spaces (as in GamePhysics, which has the abbreviation "Game Physics"). But this would require the folder name to be exactly the same as the abbreviation. 

Do you think I should keep the `directoryNames` for all the examples or modify their abbreviations to match the folder names and use those instead?